### PR TITLE
[Validator] added missing Polish translation

### PR DESCRIPTION
**Pull request by thunderer**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

Added missing translation. I'm a native Polish speaker.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 1]
```diff
@@ -438,6 +438,10 @@
                 <source>This value is not a valid MAC address.</source>
                 <target>Ta wartość nie jest prawidłowym adresem MAC.</target>
             </trans-unit>
+            <trans-unit id="113">
+                <source>This URL does not contain a TLD.</source>
+                <target>Podany adres URL nie zawiera TLD.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Let's not assume people know what a TLD is? GPT suggests this :)
`Ten URL nie zawiera domeny najwyższego poziomu (TLD).`

Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Sure, I didn't see other translations expanding the meaning of either URL or TLD, I'll update the PR shortly. :)

Comment (thunderer) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: @nicolas-grekas done!

**Comment by nicolas-grekas** [comment]
Thank you @thunderer.

**Pull request merged**